docs/user-guide/configure-zowe-runtime.md

```diff
@@ -381,94 +381,108 @@ TSO user IDs using Zowe must have permission to access the z/OSMF services that
+    Because the Cross Memory Server provides priviledges services, its load libraries require APF-authorization. To check whether a load library is APF-authorized, you can issue the following TSO command:
 
     ```
     D PROG,APF,DSNAME=ZWES.SISLOAD
     ```
-    where the value of DSNAME is the name of the data set that contains the ZWESIS01 load module.
+    where the value of DSNAME is the name of the data set that contains the ZWESIS01 and ZWESAUX load modules.
 
-    To dynamically add the load library to the APF list, run one of the following TSO commands:
+    To dynamically add a load library to the APF list if the load library is not SMS-managed, issue the following TSO command:
 
     ```
     SETPROG APF,ADD,DSNAME=ZWES.SISLOAD,VOLUME=volser
-    (If the load library resides on Non SMS-Managed Volume)
-    Or
+    ```
+    If the load library is SMS-managed, issue the following TSO command:
+    ```
     SETPROG APF,ADD,DSNAME=ZWES.SISLOAD,SMS
-    (If the load library is SMS-Managed library)
     ```
-    where the value of DSNAME is the name of the data set that contains the ZWESIS01 load module.
+    where the value of DSNAME is the name of the data set that contains the ZWESIS01 and ZWESAUX load modules.
 
-1. PARMLIB member
+1. Add a PARMLIB member:
 
-    The Zowe cross memory server started task requires a valid ZWESISPxx PARMLIB member to be found at startup. The file `xmem-server/files/zss/SAMPLIB/ZWESIP00` contains the default configuration values.  You can copy this member to your system PARMLIB data set, or allocate the default PDS data set ZWES.SISAMP that is specified in the ZWESIS01 started task JCL.
+    When started, the ZWESIS01 started task must find a valid ZWESISPxx PARMLIB member. The `xmem-server/files/zss/SAMPLIB/ZWESIP00` file contains the default configuration values. You can copy this member to your system PARMLIB data set, or allocate the default PDS data set ZWES.SISAMP that is specified in the ZWESIS01 started task JCL.
 
-1. Security requirements for the cross memory server
+1. Configure SAF:
 
-    The Zowe cross memory server performs a sequence of SAF checks to protect its services from unauthorized callers.  This is done by using the FACILITY class and an entry for `ZWES.IS`. Valid callers must have `READ` access to the `ZWES.IS` class. The following examples assume that you will be running the ZOWESVR STC under the IZUSVR user.
+    The Cross Memory Server performs a sequence of SAF checks to protect its services from unauthorized callers. To do this, it uses the FACILITY class and an entry for `ZWES.IS`. Valid callers must have `READ` access to the `ZWES.IS` class. To activate the FACILITY class, define a ZWES.IS profile, and grant the IZUSVR user READ access (these commands assume that you will be running the ZOWESVR STC under the IZUSVR user), use the following commands:
```

Comment:
Can we also add a statement that the user under which ZWESAUX is recommended to have READ access to ZWES.IS? I can imagine that some developers may want to call the parent cross-memory server from within their guest module in ZWESAUX.

```
The Cross Memory Server performs a sequence of SAF checks to protect its services from unauthorized callers. To do this, it uses the FACILITY class and an entry for `ZWES.IS`. Valid callers must have `READ` access to the `ZWES.IS` profile. Additionally, due to ZWESAUX being a caller of the cross-memory server, the STC user of ZWESAUX should also have `READ` access to the `ZWES.IS` profile,
To activate...
```

Reply:
If we're going to also show how to grant READ access to the user under which ZWESAUX runs we'll need to add the command. Like this?

```
PERMIT ZWES.IS CLASS(FACILITY) ID(IZUSVR) ACCESS(READ)
PERMIT ZWES.IS CLASS(FACILITY) ID(<zwesaux_user>) ACCESS(READ)
```
where <zwesaux_user> is the user under which the ZWESAUX started task runs.

Reply:
Did that, please review.